frontend/taipy-gui/src/components/Taipy/Chart.tsx

```diff
@@ -788,9 +861,89 @@ const Chart = (props: ChartProp) => {
                 }
             }
         },
-        [getRealIndex, dispatch, updateVars, propagate, props.onChange, config.traces.length, module]
+        [getRealIndex, dispatch, updateVars, propagate, props.onChange, config.traces.length, module],
     );
 
+    useEffect(() => {
+        if (!dataPl.length || !animationDataValues) return;
+
+        const toFramesData = dataPl.map((trace) => {
+            const isYaxisAnimated = Object.prototype.hasOwnProperty.call(animationDataValues, (trace as ScatterData).name);
```

Comment:
```suggestion
            const isYaxisAnimated = Object.prototype.hasOwn(animationDataValues, (trace as ScatterData).name);
```
`hasOwn()` is 'safer' than `hasOwnProperty()` and I don't understand why you need the `call()`.

Reply:
I'm also a bit worried about the use of `ScatterData` here...
If this means not all chart types can use the feature, so be it.
But then we want to ignore the animation data (and document the limitation).